sdk/maps/azure-maps-search/src/test/java/com/azure/search/MapsSearchAsyncClientTest.java

```diff
@@ -0,0 +1,4 @@
+package com.azure.search;
```

Comment:
No unfinished code in the main branch. Remove this until it can be re-added later with real code.